region/maridia/inner-green.json

```diff
@@ -1363,6 +1474,23 @@
                     ]}
                   ],
                   "devNote": "Other weapons take too long to kill the enemy."
+                },
+                {
+                  "name": "East Sand Hall Evir Freeze (Center to Left)",
+                  "notable": true,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canTrickyJump",
+                    "canTrickyUseFrozenEnemies",
+                    "canPlayInSand",
+                    "canCrouchJump",
+                    {"ammo": { "type": "PowerBomb", "count": 1 }},
+                    {"enemyDamage": { "enemy": "Evir", "type": "particle", "hits": 1 }}
+                  ],
+                  "note": [
```

Comment:
I'd like to see HiJumpless in the note.
Suitless Water rooms with any action at all probably have HiJump as an assumed given unless specified.